settings.gradle.kts

```diff
@@ -21,6 +21,7 @@ includeBuild("build-logic")
 include(
     "sentry",
     "sentry-kotlin-extensions",
+    "sentry-build-distribution",
```

Comment:
I would love to name it `sentry-android-distribution` can we do that? Historically, we add `-android` to everything android-related. And imho it's fine to omit the `-build` part here as to me `distribution` already implies builds/artifacts/apks/etc.